frontend/src/components/common/authorized-menu/components/become-a-mentor/styles.module.scss

```diff
@@ -1,14 +1,16 @@
 .mentorWrapper {
   display: flex;
+  flex-shrink: 0.5;
   flex-direction: column;
   align-items: center;
 }
 
 .imageWrapper {
-  transform: translateY(70px);
+  position: absolute;
```

Comment:
```suggestion
```
lets use `transform` (but only for the image) and margin-bottom to create a space between image and button